pkg/machines/scripts/create_machine.sh

```diff
@@ -9,6 +9,9 @@ OS="$5"
 MEMORY_SIZE="$6" # in MiB
 STORAGE_SIZE="$7" # in GiB
 START_VM="$8"
+CHOOSE_STORAGE="$9"
```

Comment:
You don't need to pass the CHOOSE_STORAGE. In case STORAGE_POOL is undefined just let virt-install choose one for you.

Reply:
I'm quite not sure about that. It would be quite conflicting with what "source type: existing disk image". If right user checks "choose storage", but later changes source type to existing disk image, and "choosing storage option" disappears, I have to add more handling, for example handling which would set storage_pool to undefined. It would a get a bit messy.
Right now, storage_pool keeps old value, because if choose_storage is false, then value in storage_pool is not used anyway. What do you think?

But I don't have a strong opinion on that. If you believe what I'm doing right now is wrong, I will change it.

Reply:
You 're right. Didn't think of that. Keep it as is. Thanks.

Reply:
Well, in another comment you are suggesting to always have source type option "existing disk image" at hand, and even if user chooses destination storage, if he later chooses source as existing disk image, then choose destination storage option will disappear right?
For that, I need to add handling, which will set choose_storage to false and storage_pool to undefined.

Reply:
You can ignore the storage_pool I guess and just reset the choose_storage.